mind_matter_api/api/emergency_contacts.py

```
@@ -0,0 +1,98 @@
+from flask import jsonify, request, current_app
```

Ruff (0.8.2)

`flask.current_app` imported but unused

Remove unused import: `flask.current_app`

(F401)